src/librustc_error_codes/error_codes/E0224.md

```diff
@@ -0,0 +1,17 @@
+This fails because you are using dynamically sized trait objects without Trait
```

Comment:
```suggestion
This fails because you are using dynamically sized trait objects without specifying a trait
```

Reply:
I'd reword it to (integrating @Dylan-DPC suggestion): "A dynamically sized trait was used without specifying a trait".